site.css

```
@@ -0,0 +1,1352 @@
+  --paper: #f4f7fb;
+  --mist: #9aa7b8;
+  --dim: #667284;
+  --line: rgba(207, 226, 255, 0.14);
+  --line-bright: rgba(221, 237, 255, 0.3);
+  --signal: #a9d5ff;
+  --signal-bright: #e0f2ff;
+  --glass: rgba(12, 18, 27, 0.48);
+  --ease-out: cubic-bezier(0.23, 1, 0.32, 1);
+  --ease-in-out: cubic-bezier(0.77, 0, 0.175, 1);
+  --sans: "Manrope", "Helvetica Neue", sans-serif;
+  --serif: "Instrument Serif", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)